cmd/cmd.go

```diff
@@ -554,6 +554,19 @@ var decodeEblock = &cobra.Command{
 		}
 		g.Config = Config
 
+		if height, err := strconv.Atoi(args[0]); err == nil {
+			// fetch the eblock at the height
+			dblock, _, err := factom.GetDBlockByHeight(int64(height))
+			if err != nil {
+				CmdError(cmd, err)
+			}
+			for _, ent := range dblock.DBEntries {
+				if ent.ChainID == "a642a8674f46696cc47fdb6b65f9c87b2a19c5ea8123b3d2f0c13b6f33a9d5ef" {
+					args[0] = ent.KeyMR
+				}
+			}
```

Comment:
if there is no eblock found we should error-exit here, though this is an extreme edge case

Reply:
It will break anyway. This is mainly a debug function, and if the eblock is not found, the args[0] is set to the number, which will break on the next use of it. I can make it a cleaner exit though